### PR TITLE
[Bug][Calling] Fix network manager

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/Podfile.lock [hunk 1]
```diff
@@ -83,4 +83,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 1ff305af525c1895ae27d267c25e47a4c44abee7
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.12.1
```
Comment (Mohtasim) [inline AzureCommunicationUI/Podfile.lock:86]: can you avoid checking in pod file?

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/Manager/NetworkManager.swift [hunk 2]
```diff
@@ -18,10 +19,13 @@ class NetworkManager: NetworkManagerProtocol {
 
     init() {
         networkMonitorQueue = DispatchQueue(label: Constant.networkQueue)
-    }
-
-    func isOnline() -> Bool {
-        return monitor?.currentPath.status == .satisfied
+        monitor?.pathUpdateHandler = { [weak self] path in
```
Comment (pavelprystinka) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/Manager/NetworkManager.swift:22]: Could you please move this to `startMonitor()`
Also, do we even need to `init()` ? Can `networkMonitorQueue = DispatchQueue(label: Constant.networkQueue)` be moved to `startMonitor()`?

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

**Review by Mohtasim — DISMISSED** [review]
Please change the base branch to calling-1.3.1

**Review by pavelprystinka — DISMISSED** [review]

**Review by Mohtasim — DISMISSED** [review]

**Comment by ShaunaSong** [comment]
Closed, change the based branch.

**Pull request closed without merging**